testing/dynamodb/integration_test.go

```diff
@@ -91,6 +113,12 @@ func initIntegrationTests() (cleanup func()) {
 		log.Fatalf("cannot create databaseAdmin client: %v", err)
 	}
 
+	cfg := aws.Config{
+		Endpoint: aws.String("http://localhost:8000"),
```

Comment:
Does this integration test run against a local emulator?

Reply:
Correct it uses the amazon/dynamodb-local image.